cli/searchflags/flags.go

```diff
@@ -0,0 +1,58 @@
+package searchflags
+
+import (
+	"errors"
+	"flag"
+	"math"
+
+	"github.com/brimsec/zq/pkg/nano"
+)
+
+type ts nano.Ts
```

Comment:
Since we use `ts` sometimes to refer to a variable of type nano.Ts, could you use a different name?